src/otx/recipe/_base_/data/mmaction_base.yaml

```diff
@@ -5,7 +5,7 @@ config:
   mem_cache_img_max_size:
     - 500
     - 500
-  image_color_channel: RGB
+  image_color_channel: BGR
```

Comment:
Why do we have to use BGR? Is it because the pretrained weights expect BGR?

Reply:
Yes

Reply:
I changed it to RGB [792ac01](https://github.com/openvinotoolkit/training_extensions/pull/2961/commits/792ac01e4abf151f20667610dfb840783cab365d)

Reply:
> I changed it to RGB [792ac01](https://github.com/openvinotoolkit/training_extensions/pull/2961/commits/792ac01e4abf151f20667610dfb840783cab365d)

I see, anyway, it's up to you, Maybe RGB would hurt the final accuracy, if the weights expect BGR